sources/packages/load-test/src/utils/sims-api/sims-api.ts

```diff
@@ -0,0 +1,59 @@
+import http, { RefinedResponse, ResponseType } from "k6/http";
+import {
+  BASE_API_URL,
+  KEY_CLOAK_TOKEN_URL,
+  INSTITUTION_ADMIN_USER,
+  INSTITUTION_ADMIN_PASSWORD,
+} from "../../../config.env";
+import { UserPasswordCredential, getCachedToken } from "../auth";
+
+/**
+ * Authorized parties recognized by the API.
+ */
+export enum AuthorizedParties {
+  Institution = "institution",
+  Student = "student",
+  AEST = "aest",
+  SupportingUsers = "supporting-users",
+}
+
+/**
+ * Get the SIMS API token.
+ * @returns SIMS API token.
+ */
+export function getAPIToken(authorizedParty: AuthorizedParties): string {
```

Comment:
do we need this method? is it not same as  `getToken()` in token-helpers? let me know if I am missing something.

![image](https://github.com/bcgov/SIMS/assets/54600590/e0e4d693-df40-4335-8dca-454ea5611ff0)

Reply:
same qn

Reply:
Nice catch, left over from a refactor, it will be removed 😉

Reply:
Code updated.